rules/precompiled_apple_resource_bundle.bzl

```diff
@@ -199,6 +199,12 @@ def _precompiled_apple_resource_bundle_impl(ctx):
     return [
         AppleResourceInfo(
             unowned_resources = depset(),
+            datamodels = [
```

Comment:
Is it fair to say this change causes duplicating the data in the action graph or duplicating inputs somehow? Maybe we could move the `xcdatamodel` to be exclusively fed here or somehow mitigate copying it 2 times - that might cause an issue with upstream `rules_apple` but better to not have to `copy` files in general VS clone them